Add Screenshot Images to BPUG

JillTempelmeyer opened the pull request:
I'm not overly-happy with some of the blurry images... trying to work to fix these.

JillTempelmeyer committed:
Add Screenshot Images to BPUG

I'm not overly-happy with some of the blurry images... trying to work to fix these.

LeslieLundquist commented:
@JillTempelmeyer It is a good start. Yes, not sure why the screen images are so blurry. On the Mac, it is command-shift-4 and then you can drag out to select a portion of the screen to capture. Had you tried that one? They usually turn out well for me.

JillTempelmeyer commented:
Yes, that's typically how I take screen shots as well, and I took them in full-screen mode. I suspect I know why they're low resolution, though-- these are based on the BOX demo videos, as my "Public" Box Panel doesn't allow me to stage an environment to include several features discussed in this doc. 

In "Public" Box Panel, I do, however, have access to the support and chat features. I'm going through these to add/replace the low-quality images with higher-quality ones. Also, I'm leaning toward deleting the blurry images for now, as these look pretty unprofessional to me... does that make sense to you?

JillTempelmeyer committed:
Update Support Images

I left in all screenshots for now, and fixed the resolution on a couple of them. I will try to find a resource for getting the rest of these tomorrow (I think Tom Spoonemore is still in Austin tomorrow... if there's anyone you have in mind, please let me know!)
